azure-pipelines-deploy-template.yml

```diff
@@ -17,8 +20,7 @@ parameters:
   warmupPingPath: '/check'
   warmupPingStatus: '200'
   railsEnv: 'production'
-  domain:
-  staging_domain:
+  authorizedHosts:
```

Comment:
Should this be `authorisedHosts` (`s` instead of `z`)?

Reply:
Yep, that's my mistake, I'll correct that.